wsr-tic-tac-toe/tsconfig.json

```diff
@@ -68,5 +69,8 @@
     /* Advanced Options */
     "skipLibCheck": true,                     /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true  /* Disallow inconsistently-cased references to the same file. */
-  }
+  },
+  "exclude": ["node_modules", "**/*.spec.ts", "**/*.test.ts"],
```

Comment:
why spec/test. excluded?

Reply:
I had some trouble with vscode noticing jest methods, so I tried anything I could to get it to work :D Will remove this line.